quickstarts/ajax/config.yml

```diff
@@ -0,0 +1,57 @@
+id: Ajax
```

Comment:
Lets remove the id field here, this will automatically get a uuid assigned after its first published.
I think this is what is causing the quickstart validation to fail right now